tgui/packages/tgui/interfaces/Pump.tsx

```diff
@@ -0,0 +1,100 @@
+import { BooleanLike } from 'common/react';
+import { useBackend } from '../backend';
+import {
+  Button,
+  Section,
+  NumberInput,
+  LabeledList,
+  ProgressBar,
+} from '../components';
+import { Window } from '../layouts';
+
+type Data = {
+  on: BooleanLike;
+  pressure_set: number;
+  max_pressure: number;
+  last_flow_rate: number;
+  last_power_draw: number;
+  max_power_draw: number;
+};
+
+export const Pump = (props, context) => {
+  const { act, data } = useBackend<Data>(context);
+  const {
+    on,
+    pressure_set,
+    max_pressure,
+    last_flow_rate,
+    last_power_draw,
+    max_power_draw,
+  } = data;
+
+  return (
+    <Window width={330} height={200}>
+      <Window.Content>
+        <Section>
```

Comment:
```suggestion
      <Window.Content>
        <Section>
```
Оберни секцию в Stack и Stack.Item
Стаку пропиши fill vertical
Стак айтему grow
секции fill

Reply:
поставил только филл, как ты и попросил xdd